### PR TITLE
Simplify script

### PR DESCRIPTION
**Pull request by isobit**

### CODE REVIEW

**Comment by aengelas** [comment]
Instead of renaming the orb file, why not just add the `# vi: set ft=yaml` line for an editor hint? Doesn't really matter though.

**Comment by isobit** [comment]
> Instead of renaming the orb file, why not just add the `# vi: set ft=yaml` line for an editor hint? Doesn't really matter though.

True that would work, I figured `.tmpl.yml` was more correct anyway though, and hopefully would work with other editors as well.

File: orb.tmpl.yml [hunk 1]
```diff
@@ -30,10 +30,7 @@ jobs:
...
       - setup_remote_docker:
-          version: 20.10.14
+          version: 20.10.18
       - run:
           name: Build, tag, and push to Docker Hub and AWS ECR
-          command: docker-build --ecr -- << parameters.build-args >>
-      - run:
-          name: Ensure ECR lifecycle policy is installed
-          command: docker-build aws-ecr-put-default-lifecycle-policy
+          command: docker-build << parameters.build-args >>
```
Comment (aengelas) [inline orb.tmpl.yml:36]: So it looks like the --ecr behavior is now the default, right? Assuming so, looks good.

**Review by aengelas — APPROVED** [review]

File: orb.tmpl.yml (hunk 1 above)
Comment (isobit) [reply orb.tmpl.yml:36]: Indeed yeah, it just does all that by default with no opt-out.

**Pull request merged**